app/_how-tos/filter-requests-based-on-header-names.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Filter requests based on header names
+content_type: how_to
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+    - konnect
+
+tools:
+    - deck
+
+prereqs:
+  entities:
+    services:
+        - example-service
+    routes:
+        - example-route
+
+min_version:
+  gateway: '3.4'
+
+plugins:
+  - pre-function
+
+entities:
+  - service
+  - route
+  - plugin
+
+tags:
+  - routing
+  - serverless
+
+tldr:
+  q: How do I filter which requests are allowed to pass through based on the presence of a header?
+  a: |
+   You can use the serverless Pre-Function plugin to detect headers in a request, and either let the request through or terminate it. 
+   
+   In this tutorial, we'll enable the Pre-Function plugin in the access phase, where it will look for a request with the header `X-Custom-Auth`.
```

Comment:
```suggestion
   In this tutorial, we'll enable the Pre-Function plugin in the `access` phase, where it will look for a request with the header `X-Custom-Auth`.
```
Just to clarify that this is the parameter they're looking for in the plugin config. Side note, I really like how you wrote this and included a bit about what it's doing.